proto/interchain_security/ccv/provider/v1/provider.proto

```diff
@@ -105,7 +105,7 @@ message GlobalSlashEntry {
   // This field is used to obtain validator power in HandleThrottleQueues.
   // 
   // This field is not used in the store key, but is persisted in value bytes, see QueueGlobalSlashEntry.
-  ProviderConsAddress provider_val_cons_addr = 4;
+  bytes provider_val_cons_addr = 4;
```

Comment:
See [diff between this branch and ICS v1.0.0](https://github.com/cosmos/interchain-security/compare/v1.0.0...shawn/partially-revert-key-assignment-type-safety), for the `proto/interchain_security/ccv/provider/v1/genesis.proto` and `proto/interchain_security/ccv/provider/v1/provider.proto` files